src/status_im/chat/models/images.cljs

```diff
@@ -159,7 +159,9 @@
 (rf/defn clear-sending-images
   {:events [:chat.ui/clear-sending-images]}
   [{:keys [db]} current-chat-id]
-  {:db (update-in db [:chat/inputs current-chat-id :metadata] assoc :sending-image {})})
+  {:db (-> db
+           (update-in [:chat/inputs current-chat-id :metadata] assoc :sending-image {})
+           (update-in [:chat/inputs current-chat-id :metadata] assoc :selected-photos))})
```

Comment:
it's better to dissoc, or assoc nil, not just assoc with key, and also you could do it in one update-in

`(update-in db [:chat/inputs current-chat-id :metadata] assoc :sending-image {} :selected-photos nil)`